packaging/makeself/install-alpine-packages.sh

```diff
@@ -42,5 +42,7 @@ apk add --no-cache -U \
   wget \
   xz \
   zlib-dev \
-  zlib-static ||
-  exit 1
+  zlib-static \
+  libunwind \
+  libunwind-static \
```

Comment:
Needs `libunwind-dev`, not `libunwind`. Also, this should be done in https://github.com/netdata/helper-images/blob/master/static-builder/Dockerfile.v1, not this file (we preload everything whenever possible so that we don’t end up wasting time at build time handling packages).

Reply:
I updated it. Also ncurses-dev seems to be missing from the list.
Please take care of the image yourself.